src/index.html

```diff
@@ -1,13 +1,47 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg"
+      alt="iMac"
+      class="card__image">
```

Comment:
Alt can be more descriptive, also indentation
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27" Retina 5K Monoblock"
        class="card__image"
      >
```

Reply:
Check the html formatting part: https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules